src/styles/style.css

```diff
@@ -0,0 +1,7 @@
+@import './blocks/stars.css';
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
Don't use the `*` selector as it impacts performance. It's better to set styles explicitly for elements that require them. Instead of using `*` to apply a universal box model, consider setting `box-sizing: border-box;` on specific elements or using inheritance like `html { box-sizing: border-box; } *, *::before, *::after { box-sizing: inherit; }`.